packages/app/src/device-info/services/device-info.service.ts

```diff
@@ -30,4 +33,37 @@ export class DeviceInfoService {
       return Result.failed(new AppError("", ""))
     }
   }
+
+  public async getDeviceFreeSpace(): Promise<ResultObject<number, string>> {
+    try {
+      const response = await this.deviceManager.device.request<
+        DeviceInfoRaw | NotSupportedDeviceInfo
+      >({
+        endpoint: Endpoint.DeviceInfo,
+        method: Method.Get,
+      })
+
+      if (!response.ok) {
+        return response
+      }
```

Comment:
🟢 
What do you think to move this part of the code to a private method (example of name `getDeviceInfoRequest`) and sharing it between `getDeviceFreeSpace` and `getDeviceInfo` ? 🤔